EIPS/eip-####.md

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
```suggestion
---
eip: 7807
```
Assigning next sequential EIP/ERC/RIP number..

Please also update the filename.